src/main/op/OP.ts

```diff
@@ -14,13 +16,17 @@ import {
   ParsedAuthorizationRequestURI,
   ResponseMode,
   SIOPErrors,
+  SIOPResonse,
   UrlEncodingFormat,
   VerifiedAuthorizationRequest,
 } from '../types';
+import { AuthorizationEvents } from '../types/Events';
 
 import { Builder } from './Builder';
 import { createResponseOptsFromBuilderOrExistingOpts, createVerifyRequestOptsFromBuilderOrExistingOpts } from './Opts';
 
+export const opEventEmitter = new EventEmitter();
```

Comment:
Why not create this in the OP? You then have a new emitter per OP

Reply:
changed